src/gtk/dataview.cpp

```diff
@@ -4630,6 +4632,39 @@ gtk_dataview_motion_notify_callback( GtkWidget *WXUNUSED(widget),
     return FALSE;
 }
 
+//-----------------------------------------------------------------------------
+// "drag_data_get" signal
+//-----------------------------------------------------------------------------
+
+static void gtk_dataview_drag_data_get_callback(GtkWidget *widget,
+    GdkDragContext *WXUNUSED(context), GtkSelectionData *selection_data,
+    guint WXUNUSED(i), guint WXUNUSED(t))
+{
+    GtkTreeView *tree = (GtkTreeView*)widget;
+    GtkWxTreeModel *wxtree_model = (GtkWxTreeModel *) gtk_tree_view_get_model(tree);
+    GtkTreeSelection *s = gtk_tree_view_get_selection(tree);
+    GtkTreeIter it;
+    gtk_tree_selection_get_selected(s, NULL, &it);
+    GtkTreePath *path = gtk_tree_model_get_path((GtkTreeModel*)wxtree_model,&it);
+    wxtree_model->internal->drag_data_get((GtkTreeDragSource*)1, path, selection_data );
```

Comment:
Could you please explain why do we pass `1` here (and also below)? What does it mean?